### PR TITLE
Fixed projections with multiple interceptors.

### PR DESCRIPTION
**Pull request by PascalSenn**
Previously it was not possible to use multiple interceptors.
Nested resolvers like the following were not possible. 

```csharp
public class Foo 
{
   [UseFiltering]
   [UseSorting]
   public List<Bar> Nested { get; set; }
}
```

resolves: #2767

### CODE REVIEW

File: src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        {
+            if (_first is IProjectionFieldInterceptor<T> firstHandler)
+            {
+                firstHandler.AfterProjection(context, selection);
+            }
+
+            if (_second is IProjectionFieldInterceptor<T> secondHandler)
+            {
+                secondHandler.AfterProjection(context, selection);
+            }
```
Comment (PascalSenn) [inline src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs:50]: Should I flip these @michaelstaib ?

Comment (PascalSenn) [reply src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs:50]: Like 
BeforeProjection: first, second
AfterProjection: second, first

Comment (PascalSenn) [reply src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs:50]: I will also rename second to next in this case

Comment (michaelstaib) [reply src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs:50]: yes, seems more logical ....

Comment (michaelstaib) [reply src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs:50]: ??

Like
BeforeProjection: first, second
AfterProjection: second, first

Comment (michaelstaib) [reply src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs:50]: Didn't you want to flip?

Comment (PascalSenn) [reply src/HotChocolate/Data/src/Data/Projections/Convention/ProjectionInterceptorCombinator.cs:50]: i do, thought i did commit..
maybe did not push

**Review by michaelstaib — APPROVED** [review]

**Pull request merged**